frontend/public/components/operator-hub/operator-hub-items.tsx

```diff
@@ -77,31 +79,24 @@ const providerSort = provider => {
   return provider.value;
 };
 
-const providerTypeSort = provider => {
-  switch (provider.value) {
-    case 'Red Hat':
-      return 0;
-    case 'Certified':
-      return 1;
-    case 'Community':
-      return 2;
-    case 'Custom':
-      return 4;
-    default:
-      return 5;
-  }
-};
-
-const installedStateSort = provider =>{
-  switch (provider.value) {
-    case 'Installed':
-      return 0;
-    case 'Not Installed':
-      return 1;
-    default:
-      return 3;
-  }
-};
+enum providerTypeSort {
```

Comment:
I don't think we want an enum for the sort value, but rather the provider types themselves. So something like

```ts
enum ProviderType {
  'Red Hat',
  'Certified',
  'Community',
  'Custom',
}
```

Then keep the sort logic as before, just checking the enum values.

Reply:
Updated, thanks.